ChangeLog

```diff
@@ -1,3 +1,12 @@
+Changes in squid-5.4 (07 Feb 2021):
+
+	- Bug 5189: Preserve configured order of intermediate CA certificate chain
+	- Bug 5188: Fix reconfiguration leaking tls-cert=... memory
+	- Bug 5187: Properly track (and mark) truncated store entries
+	- Bug 5134: assertion failed: Transients.cc:221: "old == e"
+	- Bug 5132: Close the tunnel if to-server conn closes after client
+	- langpack: Fix typo in Russian texts
```

Comment:
```suggestion
	- langpack: Fix typo in Russian texts
	- copyright years and CONTRIBUTORS update
```

Since copyright years change is significant (both in terms of the number of changed lines and some patch conflicts), and CONTRIBUTORS update is controversial, I would list both. I do not insist on any of this, of course.